tests/nimdoc/t17835.nim

```diff
@@ -0,0 +1,15 @@
+discard """
+  cmd: "nim doc $file"
+  action: "compile"
+  nimout: "t17835.nim(10, 3) Warning: runnableExamples must appear before the first non-comment statement [User]"
+  joinable: false
+"""
+
```

Comment:
this test should fail once https://github.com/nim-lang/Nim/issues/16993 is fixed, but ok

Reply:
yeah, it will fail I think when #16993 is fixed